tools/tasks/scriptTasks.js

```diff
@@ -36,6 +36,15 @@ module.exports = function(gulp, config) {
             .pipe(gulp.dest(OPTIONS.DIR.DIST_SCRIPTS_CLIENT));
     }
 
+    gulp.task('jquery', function() {
```

Comment:
remove this task

Reply:
Actually, I'm a little confused, why would you include a manual script tag if you've also included `jquery-ui` in the `package.json`?